client/test/execution.test.js

```diff
@@ -33,55 +31,28 @@ describe('Execution', function() {
-      '/domain/ci-test/workflows/email-daily-summaries/emailRun1/summary',
-      '/domain/ci-test/workflows/email-daily-summaries/emailRun1/history',
-      '/domain/ci-test/workflows/email-daily-summaries/emailRun1/stack-trace',
-      '/domain/ci-test/workflows/email-daily-summaries/emailRun1/queries'
-    ])
-    scenario.vm.$el.querySelector('section.execution > nav a.stack-trace').should.not.have.property("display", "none")
-    scenario.vm.$el.querySelector('section.execution > nav a.queries').should.not.have.property("display", "none")
-  })
-
-  describe('Summary', function() {
+  describe('Workflow Statistics', () => {
     it('should show statistics from the workflow', async function () {
-      var [summaryEl] = await summaryTest(this.test)
-
-      summaryEl.querySelector('.workflow-id dd').should.have.text('email-daily-summaries')
-      summaryEl.querySelector('.run-id dd').should.have.text('emailRun1')
-      summaryEl.querySelector('.history-length dd').should.have.text('14')
-      summaryEl.querySelector('.workflow-name dd').should.have.text('CIDemoWorkflow')
-      summaryEl.querySelector('.task-list dd a[href]')
-        .should.have.text('ci_task_list')
-        .and.have.attr('href', '/domain/ci-test/task-lists/ci_task_list')
-      summaryEl.querySelector('.started-at dd').should.have.text(moment().startOf('hour').subtract(2, 'minutes').format('dddd MMMM Do, h:mm:ss a'))
-      summaryEl.should.not.have.descendant('.close-time')
-      summaryEl.should.not.have.descendant('.pending-activities')
-      summaryEl.should.not.have.descendant('.parent-workflow')
-      summaryEl.querySelector('.workflow-status dd').should.contain.text('running')
-      summaryEl.querySelector('.workflow-status loader.bar').should.not.have.property("display", "none")
-    })
+      summaryTest(this.test)
+        .then(([summaryEl]) => {
+          summaryEl.querySelector('.workflow-id dd').should.have.text('email-daily-summaries')
+          summaryEl.querySelector('.run-id dd').should.have.text('emailRun1')
+          summaryEl.querySelector('.history-length dd').should.have.text('14')
+          summaryEl.querySelector('.workflow-name dd').should.have.text('CIDemoWorkflow')
+          summaryEl.querySelector('.task-list dd a[href]')
+            .should.have.text('ci_task_list')
+            .and.have.attr('href', '/domain/ci-test/task-lists/ci_task_list')
+          summaryEl.querySelector('.started-at dd').should.have.text(moment().startOf('hour').subtract(2, 'minutes').format('dddd MMMM Do, h:mm:ss a'))
+          summaryEl.should.not.have.descendant('.close-time')
+          summaryEl.should.not.have.descendant('.pending-activities')
+          summaryEl.should.not.have.descendant('.parent-workflow')
+          summaryEl.querySelector('.workflow-status dd').should.contain.text('running')
+          summaryEl.querySelector('.workflow-status loader.bar').should.not.have.property("display", "none")
```

Comment:
This code seems to be missing semicolons. Please include them per Uber's general coding standards. You can implement eslint rules to automatically enforce these standards, as well as other recommendations.

Reply:
Will update here and open a separate PR to introduce eslint. Thanks!